src/Data/List/NonEmpty/Extra.hs

```diff
@@ -6,7 +6,10 @@ module Data.List.NonEmpty.Extra(
     module Data.List.NonEmpty,
     (|:), (|>), snoc, (!?),
     appendl, appendr,
-    sortOn, union, unionBy,
+#if __GLASGOW_HASKELL__ < 910
+    sortOn,
+#endif
+    union, unionBy,
```

Comment:
This is an unnecessarily breaking change. Just don't make this change at all, and let it re-export `base`'s `sortOn`.

Reply:
Not so! `sortOn` is exported by the export of `module Data.List.NonEmpty` in this case.

Reply:
Ah, you're right, I missed that. I guess this doesn't hurt anything after all then.